apps/src/templates/CensusForm.jsx

```diff
@@ -96,6 +97,7 @@ class CensusForm extends Component {
             />
           )}
         </div>
+        <CensusFollowUp/>
```

Comment:
Just confirming my understanding that this component will later be shown or hidden based on the CSOptions checkbox selected above. Showing it unconditionally seems fine for now 👍

Reply:
Confirmed. It will eventually only be shown if they select that their school teaches a CS class with 20+ hours of programming